packages/fxa-auth-server/lib/senders/emails/locales/en-US.ftl

```diff
@@ -0,0 +1,5 @@
+hello-world = Hello, world!
```

Comment:
Let me know your thoughts on the directory structure to be set out for the locales